Makefile

```
@@ -378,9 +378,20 @@ test-integration: test-deps
 # Please use golangci-lint version v1.51.2 , otherwise you might encounter errors.
 # You can download version v1.51.2 at https://github.com/golangci/golangci-lint/releases/tag/v1.51.2
 .PHONY: lint
-lint:
+lint: check-linter
 	$(GOLANGCI_LINT) run --build-tags=allcomponents --timeout=20m
 
+
+################################################################################
+# Target: check-linter                                                         #
+################################################################################
+.SILENT: check-linter # Silence output other than the application run
+.PHONY: check-linter
+check-linter:
+	cd ./.build-tools && \
+	go mod tidy && \
+	go run . check-linter
```

See tests/dapr_tests.mk for an example

```suggestion
	$(RUN_BUILD_TOOLS) check-linter
```